opentracing-api/src/main/java/io/opentracing/References.java

```diff
@@ -29,6 +29,12 @@ private References(){}
     public static final String CHILD_OF = "child_of";
 
     /**
+     * Like {@link References#CHILD_OF}, but inferred via {@link Scheduler#activeContext}, not explicitly
+     * specified by the programmer (and thus less certain).
+     */
+    public static final String INFERRED_CHILD_OF = "inferred_child_of";
```

Comment:
Considering that this child reference is actually established internally by the Tracer itself, do we need a public constant? It's not like the API provides a way to query for it.